lib/widgets/content.dart

```diff
@@ -479,49 +479,51 @@ class ListNodeWidget extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     // TODO(#162): p+ul and p+ol interactions
-    final items = List.generate(node.items.length, (index) {
+    final items = List<TableRow>.generate(node.items.length, (index) {
```

Comment:
nit: Shouldn't be needed, right?
```suggestion
    final items = List.generate(node.items.length, (index) {
```

Reply:
Yes, right!